FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json

```diff
@@ -53,6 +53,34 @@
                 "description": "The name of the container to store job and data."
             }
         },
+        "filterScope":{
+            "type": "string",
+            "defaultValue": "System",
+            "metadata": {
+                "description": "The filter scope, System or Group."
+            }
+        },
+        "groupId":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The group id you want to export data for Group scope."
+            }
+        },
+        "requiredTypes":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The resources type you want to export. Will continuous export all resource types if not specified."
+            }
+        },
+        "typeFilters":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The type filters you want to filter the data. Will continuous export all data if not specified."
```

Comment:
Maybe we can update the description reference the export doc? https://docs.microsoft.com/en-us/azure/healthcare-apis/fhir/export-data

Reply:
thanks for the good suggestion, updated.